robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -162,10 +162,13 @@ public static final class ELEVATOR {
         .maxVelocity(0);
 
     public static final int ENCODER_COUNTS_PER_REV = 8196;
-    public static final double GEAR_RATIO = 50 / 14;
+    public static final double GEAR_RATIO = 3.2142857143;
```

Comment:
Where did this gear ratio come from? Based on the cad, it looks like it should be around 5.42857142857

Reply:
Zach calculated that number yesterday, the gearing in the CAD is wrong.